.github/workflows/firebase_deploy_demo.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy to Preview Channel
+
+on:
+  push:
+    branches:
+      - paypal-app-switch-feature
```

Comment:
Should we add the option to manually trigger this from any branch? I think after merge to main we still might want to re-deploy the test app